FreeRTOS_IP.c

```diff
@@ -3677,6 +3610,19 @@ uint32_t FreeRTOS_max_uint32( uint32_t a,
 }
 /*-----------------------------------------------------------*/
 
+/**
+ * @brief Get the highest value of two variables of type size_t.
+ * @param[in] a: the first value.
+ * @param[in] b: the second value.
+ * @return The highest of the two values.
+ */
+size_t FreeRTOS_max_size_t( size_t a,
```

Comment:
maybe we should make these inline?

Reply:
These functions used to have both a macro and an inline version, depending on whether `portINLINE` is defined. When we started working on MISRA compliance, we decided to make all these small helper functions global.